deny.toml

```diff
@@ -11,6 +11,9 @@
 
```

Comment:
The format of this file changed, so I just regenerated it from a fresh template and ported existing directives until it works.